alpine-fat/Dockerfile

```diff
@@ -104,6 +106,7 @@ RUN apk add --no-cache --virtual .build-deps \
         --with-lua-include=/usr/local/openresty/luajit/include/luajit-2.1 \
     && make build \
     && make install \
+    && PERL_MM_USE_DEFAULT=1 cpan install Test::Nginx \
```

Comment:
@neomantra I made these changes. Surprisingly, it bumped the image size up from 255 MB to 291 MB. If we're comfortable with that then I can squash as is. I do think the curl approach is perfectly valid, and does avoid having to install cpan.

Reply:
Hmmm... I think there might be cpan cache's that need to get deleted too --  `/root/.cpan` is taking 55.6M.   So after removing cpanminus, there needs to be this line:
`&& rm -rf /root/.cpan`

Reply:
sorry, i see you aren't using cpanminus...  just put that after the install.

Reply:
Gah, sorry again... I think it should just be the build folder?  `/root/.cpan/build`   or perhaps there's a way to tell cpan not to save stuff.   The build folder is ~18MB.

Reply:
Thanks @neomantra, I'll give this a try. I'm busy all day today but I'll pick this up again in a couple days.

Reply:
@neomantra You were spot on. Removing the `/root/.cpan` directory got the image down to 241 MB. Versus the current image that's 234 MB.